steps/representative/representative-details/RepresentativeDetails.js

```diff
@@ -1,15 +1,56 @@
-const { Question } = require('@hmcts/one-per-page');
+'use strict';
+
+const { Question, form, field, goTo } = require('@hmcts/one-per-page');
+const { postCode, firstName, lastName, whitelist, phoneNumber } = require('utils/regex');
+const Joi = require('joi');
 const urls = require('urls');
 
+const emailOptions = { minDomainAtoms: 2 };
+
 class RepresentativeDetails extends Question {
 
     get url() {
         return urls.representative.representativeDetails;
     }
 
-    get form() {}
-    
-    next() {}
+    get form() {
+        const fields = this.content.fields;
+
+        return form(
+            field('firstName')
+                .joi(fields.firstName.error.required, Joi.string().required())
+                .joi(fields.firstName.error.invalid, Joi.string().regex(firstName)),
+
+            field('lastName')
+                .joi(fields.lastName.error.required, Joi.string().required())
+                .joi(fields.lastName.error.invalid, Joi.string().regex(lastName)),
+            field('organisation'),
+
+            field('addressLine1')
+                .joi(fields.addressLine1.error.required, Joi.string().regex(whitelist).required()),
+
+            field('addressLine2'),
```

Comment:
All fields should be validated even optional ones, if a user adds something we want to ensure it's at least whitelisted.